Dockerfile

```diff
@@ -1,45 +1,58 @@
-FROM node:16.16.0-alpine3.15 AS base
```

Comment:
changes are based on the [nextjs docker examples](https://github.com/vercel/next.js/tree/canary/examples/with-docker)